js/app.js

```diff
@@ -118,7 +118,7 @@ Application = function(p)
 
 		//////////////
 
-		peertubeServer : 'https://test.peertube.pocketnet.app/api/v1/',
+		peertubeServer : 'http://65.108.83.132/api/v1/',
```

Comment:
Why not using HTTPS via port **443**? It is the same thing no.
`peertubeServer: 'https://65.108.83.132:443/api/v1/',`